bokeh/settings.py

```diff
@@ -608,8 +608,15 @@ def dev(self):
     A password to decrypt the SSL keyfile, if necessary.
     """)
 
-    strict = PrioritizedSetting("strict", "BOKEH_STRICT", convert=convert_bool, help="""
-    Whether validation checks should be strict (i.e. raise errors).
+    validation_exceptions = PrioritizedSetting("validation_exceptions", "BOKEH_VALIDATION_EXCEPTIONS", default="none", help="""
```

Comment:
After a little more thought, I think a better name would be `validation_level` / `BOKEH_VALIDATION_LEVEL`